src/Files.Uwp/Properties/Default.rd.xml

```diff
@@ -21,7 +21,8 @@
       An Assembly element with Name="*Application*" applies to all assemblies in
       the application package. The asterisks are not wildcards.
     -->
-    <Assembly Name="SQLitePCL" Dynamic="Required Public" />
+    <Type Name="SQLitePCL.Batteries_V2" Dynamic="Required Public" />
```

Comment:
What's this change for?

Reply:
Does not work in Release mode otherwise 😕